plugins/@grouparoo/mailchimp/src/lib/connect.ts

```diff
@@ -2,6 +2,12 @@ import { SimpleAppOptions } from "@grouparoo/core";
 import Mailchimp from "mailchimp-api-v3";
 
 export async function connect(appOptions: SimpleAppOptions) {
-  const client = new Mailchimp(appOptions.apiKey?.toString());
-  return client;
+  if (appOptions.apiKey) {
+    return new Mailchimp(appOptions.apiKey?.toString());
+  } else if (appOptions.oAuthToken) {
+    const [authToken, dc] = appOptions.oAuthToken.toString().split("::");
+
+    //@ts-ignore Mailchimp client typings have not been updated for OAuth support (with dc param)
```

Comment:
Made a PR for this here https://github.com/thorning/node-mailchimp/pull/78 but has not been merged in yet. Once it is, we can remove this `@ts-ignore`

Reply:
Nice!